flake.nix

```diff
@@ -191,6 +191,7 @@
                 ++ [
                   pkgs.nats-server
                   pkgs.redis
+                  pkgs.vault-bin
```

Comment:
just out of curiosity, is there any particular reason to use `-bin` version of this package and not just `pkgs.vault`?

Reply:
Nope! That's coming out -- was using that during testing